docs/grafana.rst

```diff
@@ -0,0 +1,320 @@
+where *date* and *time* are local date and time for pc where BEAM is running and the simulation name is taken from ``beam.agentsim.simulationName``
+config value. The BEAM run list containing the ``run name`` switch is updated after a page refresh. So make sure to refresh your browser after
+a new BEAM run is started and output from the simulation have begun to accumulate. If there are no active BEAM runs in the local history then one will see no choice for ‘run_name’:
+
+.. figure:: _static/figs/grafana-run-name-switch.png
+
+    BEAM run name switch
+
+Disable and enable data traces
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to disable and enable some of value traces on a graph by clicking on the name of a trace.
+
+.. figure:: _static/figs/grafana-enable-disable-traces.png
+
+    enable | disable traces
+
+Select BEAM iteration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On Iteration View it is possible to select the iteration to see through the ‘iteration_num’ switch. Iteration number switch will update after a page refresh.
+
+.. figure:: _static/figs/grafana-iteration-switch.png
+
+    BEAM iteration switch
+
+Select data update frequency
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is a switch to choose how frequent grafana should gather data from the output DB. The switch is in the top right corner in all dashboards.
+In the example below the switch is set up to update every 30 seconds. Also, there is a refresh button near the switch, this button
+does not refresh the entire page, run name or iteration num switch, *only the graphs*.
+
+.. figure:: _static/figs/grafana-update-frequency-switch.png
+
+    shoose graphs update frequency
+
+Iteration Map View
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On the **Iteration Map View** one may choose which data trace displaying as bars graph **(1)**, which data traces are displaying on a map **(2)**
+and for what hour data traces are displaying on a map **(3)**. Map is functional too, so, one may zoom in and out and move around with mouse controls.
+
+.. figure:: _static/figs/grafana-iteration-map-view.png
+
+    example of map view
+
+Configure BEAM metrics in Grafana
+------------------------------------
+
+Existing metrics
+^^^^^^^^^^^^^^^^^^
+
+To configure which metrics will be written one should use BEAM configuration files.
+
+There is a configuration entry ``beam.sim.metric.collector.metrics`` which contains names of enabled metrics
+and thus controls which metrics will be written during BEAM execution.
+
+By default metrics which do not affect performance are enabled globally in “test/input/common/metrics.conf” file.
+
+To configure metrics for specific BEAM configuration one may add the configuration entry in that specific configuration file.
+
+**Metrics description:**
+
+Metrics which contain a run name and an iteration number, **necessary for displaying any metric**::
```

Comment:
I'm not sure what was meant here. Maybe remove `,` and make it `and are`?

Reply:
the meaning was follow: these two metrics are necessary to display any other metric, so, I am going to change the text as you suggested